2535.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var differenceOfSum = function (nums) {
+  let eleSum = 0;
+  let digitSum = 0;
+
+  for (let i = 0; i < nums.length; i++) {
+    eleSum += nums[i];
+
+    while (nums[i] > 0) {
+      digitSum += nums[i] % 10;
+      nums[i] = Math.floor(nums[i] / 10);
```

Comment:
This changes the elements in `nums`, which can cause side effects.
You can create a variable/constant for `nums[i]` and manipulate it instead of manipulating the original element.